### DISCUSSION TITLE
Give an option to view starred items that aren't added to a list

### DISCUSSION THREAD

**Discussion by blacklightpy** [community/community#10982] [Other Feature Feedback, Questions, & Ideas]
Currently we can view listed items separately, but can only view starred items as a whole. If we could view starred items that don't belong to any list, we could easily categorize them.

**Comment by micah686** [comment]
I agree with this. I have a lot of starred projects, and it would be nice to see only the stars that I haven't categorized; instead of going through page after page of stars trying to find the ones you haven't added to a list yet.

**Comment by richard-hajek** [comment]
Definitely agree with this, would be a huge improvement in my opinion. I could easily categorize my whole star collection, few stars per day. But that is impossible because just searching for repos without a list takes some time
